testscripts/go_check_GUI_driver.ps1

```diff
@@ -0,0 +1,151 @@
+###############################################################################
+##
+## Description:
+## Check the vmware driver xorg-x11-drv-vmware exist in guest.
+##
+###############################################################################
+##
+## Revision:
+## V1.0 - ldu - 05/20/2019 - Check the vmware driver xorg-x11-drv-vmware exist in guest.
```

Comment:
##  v1.0.0 - ldu - 05/20/2019 - Create the script to check xorg-x11-drv-vmware

Reply:
OK